share/CWW_E_ext_more.sed

```diff
@@ -0,0 +1,278 @@
+#  E SED of Coleman, Wu, and Weedman (1980)
```

Comment:
Good point.  I will do that, thanks!  I assume I can still use the examples/data/getSEDs.py with the settings appropriately updated?

The other thing that I realized is pretty silly with the current setup is that the more thinned versions here in share/ correspond to the ones in examples/data/, yet the filenames are the same for the old (less thinned) versions in share/ and in examples/data.  Would you object to my changing the filenames of the ones in examples/data so be consistent with the more thinned ones in share/?  Or, is there any reason that demo12.py shouldn't just use the thinned versions in share/, and we can delete the copies of the SEDs from examples/data/?